src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,6 +146,18 @@ export class Dropdown extends BaseComponent<IDropdownProps, any> {
     }
   }
 
+  private _renderDropdownOption(item: IDropdownOption): JSX.Element {
```

Comment:
When this code executes, it will create a new closured function for every item. A more efficient approach is this:

```ts
public render() {
  let { onRenderItem = this._onRenderItem } = this.props;

  render (
    <ul>
      { items.map(item => (
      <li key={item.key}>
        { onRenderItem(item, this._onRenderItem) }
      </li>
      )) }
    </ul>
  );
}

@autobind
private _onRenderItem(item: IDropdownOption): JSX.Element) {
  return <span>{item.text}</span>
}